cli/cmd/compliance_aws.go

```diff
@@ -411,13 +411,85 @@ The output from status with the --json flag can be used in the body of PATCH api
 			return nil
 		},
 	}
+
+	// complianceAwsListAccountsCmd represents the list-accounts inside the aws command
+	complianceAwsSearchCmd = &cobra.Command{
+		Use:   "search <resource_arn>",
+		Short: "Search for all known violations of a given resource arn",
+		Long:  `Search for all known violations of a given resource arn.`,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			cli.StartProgress(fmt.Sprintf(" searching accounts for resource %s...", args[0]))
```

Comment:
```suggestion
			cli.StartProgress(fmt.Sprintf("Searching accounts for resource %s...", args[0]))
```

Reply:
capital, but we do want to include the space for the spinner